{"uuid":"9173d02b-8545-45e6-9c83-01dc513fb6fc","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["f80e188b-0ed3-4ef3-bd5c-e03e2a141172"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687732372022,"stop":1687732372022},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687732372251,"stop":1687732372254},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687732372392,"stop":1687732372392},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687732372489,"stop":1687732372490},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687732372508,"stop":1687732372508}],"start":1687732371891,"stop":1687732372509}